weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -90,89 +85,132 @@ def should_skip(self):
 
 class GribSplitter(FileSplitter):
 
+    def __init__(self, input_path: str, output_info: OutFileInfo,
+                 force_split: bool = False, logging_level: int = logging.INFO):
+        super().__init__(input_path, output_info,
+                         force_split, logging_level)
+        self.output_info.set_formatting_if_needed('_{typeOfLevel}_{shortName}')
+        self.split_dims = self._get_split_dims()
```

Comment:
If this is a property of `output_info` and `set_formatting_if_needed` is moved to a pure function on `output_info`, then there would be no need to overload the constructor. Indeed, the formatting in question could be static data in each Splitter class.